src/app/files/fileManager.js

```diff
@@ -166,3 +172,23 @@ class FileManager {
 }
 
 module.exports = FileManager
+
+function flatten (tree) {
```

Comment:
the param is a `tree` but above a list is used : `var fileList = Object.keys(flatten(filesList))` as input

Reply:
oh good catch. yes, i fixed it. it's now `flatten(filesTree)`